packages/deployment/app/cumulus_api.template.yml

```diff
@@ -179,15 +179,15 @@ Resources:
         Variables:
           stackName: {{../stackName}}
       {{# if this.urs}}
-        {{# if ../api_backend_url}}
-          TOKEN_REDIRECT_ENDPOINT: {{../api_backend_url}}token
+        {{# if ../parent.api_backend_url}}
+          TOKEN_REDIRECT_ENDPOINT: {{../parent.api_backend_url}}token
```

Comment:
Huh. I assume this wasn't working as is? What is the difference between now vs previously?

Reply:
now we actually grab the value out of the config file, where before, the if statement showed as false, and the redirect endpoint was the "real" one as opposed to the configured one.

I want my URLs to be explicit when I'm doing testing, and we set api_distribution_url to example.com in the defaults, but it wasn't getting picked up. 

But now that I've typed that, I added them to the backend because they weren't working for the api_distribution_url.  Is that wrong?